https://salsa.debian.org/games-team/supertuxkart/-/raw/master/debian/patches/irrlicht/use-system-libs.diff
----
Subject: [PATCH] debian/use-system-libs

Path to build irrlicht using the system libraries for png,zlib and
jpeg as well as the glext family of header files.

Signed-off-by: Christoph Egger <debian@christoph-egger.org

--- a/lib/irrlicht/source/Irrlicht/CIrrDeviceLinux.cpp
+++ b/lib/irrlicht/source/Irrlicht/CIrrDeviceLinux.cpp
@@ -35,7 +35,7 @@
 #include <GL/gl.h>
 #ifdef _IRR_OPENGL_USE_EXTPOINTER_
 #define GLX_GLXEXT_PROTOTYPES
-#include "glxext.h"
+#include <GL/glxext.h>
 #endif
 #endif
 
--- a/lib/irrlicht/source/Irrlicht/COpenGLExtensionHandler.h
+++ b/lib/irrlicht/source/Irrlicht/COpenGLExtensionHandler.h
@@ -24,7 +24,7 @@
 	#define NO_SDL_GLEXT
 	#include <SDL_video.h>
 	#include <SDL_opengl.h>
-	#include "glext.h"
+	#include <GL/glext.h>
 #elif defined(_IRR_WINDOWS_API_)
 	// include windows headers for HWND
 	#define WIN32_LEAN_AND_MEAN
@@ -34,7 +34,7 @@
 	#endif
 	#include <GL/gl.h>
 	#if defined(_IRR_OPENGL_USE_EXTPOINTER_)
-		#include "glext.h"
+		#include <GL/glext.h>
 	#endif
 	#include "wglext.h"
 
@@ -49,7 +49,7 @@
 	#endif
 	#include <OpenGL/gl.h>
 	#if defined(_IRR_OPENGL_USE_EXTPOINTER_)
-		#include "glext.h"
+		#include <GL/glext.h>
 	#endif
 #endif
 
--- a/lib/irrlicht/source/Irrlicht/COpenGLSLMaterialRenderer.h
+++ b/lib/irrlicht/source/Irrlicht/COpenGLSLMaterialRenderer.h
@@ -12,7 +12,7 @@
 	#define WIN32_LEAN_AND_MEAN
 	#include <windows.h>
 	#include <GL/gl.h>
-	#include "glext.h"
+	#include <GL/glext.h>
 #else
 #if defined(_IRR_OPENGL_USE_EXTPOINTER_)
 	#define GL_GLEXT_LEGACY 1
@@ -26,7 +26,7 @@
 	#include <GL/gl.h>
 #endif
 #if defined(_IRR_OPENGL_USE_EXTPOINTER_)
-	#include "glext.h"
+	#include <GL/glext.h>
 #endif
 #endif
 
